CMakeLists.txt

```diff
@@ -363,6 +363,58 @@ opm_add_test(test_parallelwellinfo_mpi
   NO_COMPILE
     )
 
+opm_add_test(test_parallel_wbp_sourcevalues_np1
```

Comment:
sorry for missing this earlier. this test will be a duplicate. the one processor one is added as a normal test from the CMakeLists_files.txt entry.

Reply:
> sorry for missing this earlier. this test will be a duplicate.

Ah, yes.  You're right.  I'll amend.